### PR TITLE
Change to gfile.Exists and deeper link to CIFAR-10

### PR DESCRIPTION
**Pull request by tfboyd**

### CODE REVIEW

File: official/resnet/cifar10_main.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from absl import app as absl_app
 from absl import flags
 import tensorflow as tf  # pylint: disable=g-bad-import-order
+from tensorflow.python.platform import gfile
```
Comment (robieta) [inline official/resnet/cifar10_main.py]: Drive by comment: this is exported with tf. (so you can do `tf.gfile.Exists(...)`)

Comment (tfboyd) [reply official/resnet/cifar10_main.py]: ahhh sweet I will swap that out.

Comment (tfboyd) [reply official/resnet/cifar10_main.py]: HIGH FIVE drive-by

Comment (tfboyd) [reply official/resnet/cifar10_main.py]: fixed

**Pull request merged**